### PR TITLE
Updated parameters puppet template

### PR DESCRIPTION
**Pull request by erikaheidi**
Updated the parameters.yml template for the puppet / vagrant setup.

### CODE REVIEW

File: puppet/modules/cobase/templates/symfony/parameters.yml.erb [hunk 1]
```diff
@@ -32,4 +32,22 @@ parameters:
...
 
     # Secret for the passwords
-    secret:            <%= @secret %>
\ No newline at end of file
+    secret:            <%= @secret %>
+
+    # How many posts do we want to see per page
+    posts_per_page:    10
+
+    # How many subscription do we want to see in front page
```
Comment (cordoval) [inline puppet/modules/cobase/templates/symfony/parameters.yml.erb:40]: subscriptions

File: puppet/modules/cobase/templates/symfony/parameters.yml.erb [hunk 2]
```diff
@@ -32,4 +32,22 @@ parameters:
...
+    # How many groups do we want to see in group listing
+    groups_per_page: 20
+
+# twig section
+twig:
+
+    # global variables for twig templates
+    globals:
+
+        # Enable "Fork Me On Github" ribbon pn page
```
Comment (cordoval) [inline puppet/modules/cobase/templates/symfony/parameters.yml.erb:52]: what is pn?

File: puppet/modules/cobase/templates/symfony/parameters.yml.erb [hunk 3]
```diff
@@ -32,4 +32,22 @@ parameters:
...
+        show_forkme: true
```
Comment (cordoval) [inline puppet/modules/cobase/templates/symfony/parameters.yml.erb:53]: no new line at the eof

File: puppet/modules/cobase/templates/symfony/parameters.yml.erb (hunk 1 above)
Comment (erikaheidi) [inline puppet/modules/cobase/templates/symfony/parameters.yml.erb:40]: I just copied the original parameters.yml-dist file, this is the puppet template. I can fix it here, but it must be changed there too =)

File: puppet/modules/cobase/templates/symfony/parameters.yml.erb (hunk 2 above)
Comment (erikaheidi) [inline puppet/modules/cobase/templates/symfony/parameters.yml.erb:52]: typos from the original parameters.yml

**Comment by erikaheidi** [comment]
Another thing for discussion: the box in the current Vagrantfile (the one with guest additions) is always giving download errors, I think we should change for another box from the official Vagrant repos, because these ones hosted outside (Dropbox in this case) are not trustable.

**Comment by artur-gajewski** [comment]
@erikaheidi you can choose which box, but can you make sure it contains the guest additions? This box worked fine on my laptop but I understand if others are having problem with it.

**Pull request merged**